io/zenoh-transport/src/common/pipeline.rs

```diff
@@ -235,6 +235,10 @@ impl StageIn {
                                     // Still no available batch.
                                     // Restore the sequence number and drop the message
                                     $($restore_sn)?
+                                    tracing::trace!(
+                                        "Zenoh message dropped because it's over the deadline {:?}",
```

Comment:
Print as well `msg` so there is the information which message has been dropped.

Reply:
Sounds good to me